compiler/q-implant/src/QImplant.cpp

```diff
@@ -143,6 +149,20 @@ void set_quantized_dimension(luci::CircleNode *node, const uint32_t quantized_di
   node->quantparam()->quantized_dimension = quantized_dimension;
 }
 
+void set_quantparam(luci::CircleNode *origin, luci::CircleNode *dest)
```

Comment:
```suggestion
void copy_quantparam(const luci::CircleNode *src, luci::CircleNode *dest)
```

Reply:
Found `copy_quantparam` on  [`CircleQuantParam.cpp`](https://github.com/Samsung/ONE/blob/6c77165d79bedb5fe4e4a3c09a1f4dafe99f58bd/compiler/luci/lang/src/CircleQuantParam.cpp#L28)

It's purpose is same, and to avoid duplicated code mentioned before,
removed it from `q-implant` and use `copy_quantparam` from `CircleQuantParam`